games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -18,8 +18,8 @@ RDEPEND="media-libs/libsdl[joystick,opengl,sound,video]
 	sys-libs/zlib
 	virtual/opengl
 	dev-qt/qtcore:4
-	dev-qt/qtgui:4
-	dev-qt/qtopengl:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
```

Comment:
Well, it does :). But I'm not sure if that's virtualjaguar's fault or Qt/EGL/video card problem. @damex's getting some rubbish static on the screen, I'm getting:

    QEglContext::swapBuffers(): "Bad surface (0x300D)" 
    QEglContext::swapBuffers(): "Bad surface (0x300D)" 
    QEglContext::swapBuffers(): "Bad surface (0x300D)" 
    QEglContext::swapBuffers(): "Bad surface (0x300D)" 
    QEglContext::swapBuffers(): "Bad surface (0x300D)"